steps/sonarqube-msbuild-scanner/1.0.0/step.yml

```diff
@@ -0,0 +1,113 @@
+title: SonarQube MSBuild Scanner
+summary: |
+  Step for running SonarQube MSBuild Scanner CLI.
+description: |
+  Step for running SonarQube MSBuild Scanner CLI.
+website: https://github.com/tbrushwyler/bitrise-step-sonar-qube-msbuild-scanner
+source_code_url: https://github.com/tbrushwyler/bitrise-step-sonar-qube-msbuild-scanner
+support_url: https://github.com/tbrushwyler/bitrise-step-sonar-qube-msbuild-scanner/issues
+published_at: 2018-12-11T15:47:20.764047285-05:00
+source:
+  git: https://github.com/tbrushwyler/bitrise-step-sonarqube-msbuild-scanner.git
+  commit: 130fd867ccf98641413eb79703925cdfaa551f8d
+host_os_tags:
+- osx-10.10
+- ubuntu-16.04
+project_type_tags:
+- xamarin
+type_tags:
+- utility
+toolkit:
```

Comment:
If toolkit is omitted, the bash toolkit with step.sh entry file will be the default.